pkg/job/job.go

```diff
@@ -589,3 +624,64 @@ type InputItem struct {
 	RedirectApprovalURL string
 	Inputs              []ParameterDefinition
 }
+
+// PipelineBuild represents a build detail of Pipeline.
+type PipelineBuild struct {
+	Class string `json:"_class,omitempty" description:"It’s a fully qualified name and is an identifier of the producer of this resource's capability."`
+	Links struct {
```

Comment:
Do we really need all those fields? Many fields of those come from Java objects. Please only keep the useful ones.

Reply:
As a library, we should minimize the loss of information. So I will change type of `Links` and `Causes` to `interface{}`. What about it?